app/src/main/java/org/jellyfin/androidtv/ui/startup/EditUserScreen.kt

```diff
@@ -0,0 +1,66 @@
+package org.jellyfin.androidtv.ui.startup
+
+import androidx.preference.PreferenceCategory
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.auth.AccountManagerHelper
+import org.jellyfin.androidtv.auth.AuthenticationStore
+import org.jellyfin.androidtv.ui.preference.dsl.OptionsFragment
+import org.jellyfin.androidtv.ui.preference.dsl.optionsScreen
+import org.koin.android.ext.android.get
+import java.util.*
+
+class EditUserScreen : OptionsFragment() {
+
+	override val screen by optionsScreen {
+		val serverUUID = requireArguments().get(ARG_SERVER_UUID)
+		val userUUID = requireArguments().get(ARG_USER_UUID)
+
+		if (serverUUID !is UUID || userUUID !is UUID)
+			return@optionsScreen
+
+		val accountManagerHelper = get<AccountManagerHelper>()
+		val authenticationStore = get<AuthenticationStore>()
+		val server = authenticationStore.getServer(serverUUID) ?: return@optionsScreen
+
+		val user = authenticationStore.getUser(serverUUID, userUUID) ?: return@optionsScreen
+		title = context?.getString(R.string.lbl_user_server, user.name, server.name)
+
+		val account = accountManagerHelper.getAccount(userUUID)
+
+		when {
+			account?.accessToken != null -> {
+				action {
+					setTitle(R.string.lbl_sign_out)
+
+					setListener { pref ->
+						accountManagerHelper.removeAccount(account)
+
+						(pref.parent as PreferenceCategory).removePreference(pref)
+
+						return@setListener true
+					}
+				}
+			}
+		}
+
+		action {
+			setTitle(R.string.lbl_remove)
+
+			setListener { _ ->
+				authenticationStore.removeUser(serverUUID, userUUID)
+
+				//Pop the back stack manually
+				//Do it twice because the EditServerScreen doesn't update to remove the listed user we just deleted
+				requireActivity().supportFragmentManager.fragments[0].childFragmentManager.popBackStackImmediate()
```

Comment:
How can we be certain that the first element in `fragments` is the fragment we want to pop?